frontend/src/components/DevicesBrowser.vue

```diff
@@ -342,6 +359,9 @@ export default {
         clearInterval(this.checkInterval)
     },
     methods: {
+        applyFilter (filter) {
+            this.filter = filter
```

Comment:
I found the spec of this `filter` object unclear, and had to dig into `DeviceStatusBar` to understand how it works. 

Does the DevicesBrowser need the `this.property`, `bucket` on top of devices? Or could the logic for those two properties remain within the two DeviceStatusBars, making the event something like:

```
applyFilter (devices) {
            // logic to filter this.devices to only include devices
}
```

Reply:
> Or could the logic for those two properties remain within the two DeviceStatusBars

The reason for this was for when a user switches between two filters, the original filter needs to be reset. The parent (`DevicesBrowser`) just acts as a broker to communicate the `property` and `bucket` data between the different `DeviceStatusBar` components

I will add comments accordingly to make this more explicit